app/views/audit_mailer/summary.text.erb

```diff
@@ -0,0 +1,15 @@
+<%= _("Foreman audit summary") %>
+=================================
+<% if @audits.size > 0 %>
+  <%= n_('Displaying %{num_audits} of %{total_audits} audits',
+         'Displaying %{num_audits} of %{total_audits} audits', @count) % {:num_audits => @limit, :total_audits => @count} %>
```

Comment:
num_audits seems to be blank for me in both text and HTML mailers, I see `Displaying  of 1 audits` but I can't tell why.  Any idea?